README.md

```diff
@@ -6,13 +6,15 @@
 
 * * *
 
-<img width="64" src="https://raw.githubusercontent.com/laurent22/joplin/master/Assets/LinuxIcons/256x256.png" align="left" /> **Joplin** is a free, open source note taking and to-do application, which can handle a large number of notes organised into notebooks. The notes are searchable, can be copied, tagged and modified either from the applications directly or from your own text editor. The notes are in [Markdown format](#markdown).
+<img width="64" src="https://raw.githubusercontent.com/laurent22/joplin/master/Assets/LinuxIcons/256x256.png" align="left" /> **Joplin** is a free, open source note taking and to-do list application available for Windows, Linux, macOS, Android, and iOS. It can easily handle a vast number of notes, which are organised into notebooks. Notes can be searched, copied, tagged, and edited -- even with your favourite text [editor](#External-text-editor).
 
-Notes exported from Evernote via .enex files [can be imported](#importing) into Joplin, including the formatted content (which is converted to Markdown), resources (images, attachments, etc.) and complete metadata (geolocation, updated time, created time, etc.). Plain Markdown files can also be imported.
+Notes can be easily imported from Evernote into Joplin using [.enex files](#importing). Formatted content, resources (images, attachments, etc.), and complete metadata is included in the import, which is stored in [Markdown](#markdown) format. Plain Markdown files can also be imported, increasing flexibility and usability.
```

Comment:
Starting off with 'can be easily imported from Evernote' instead of 'exported from Evernote' focuses the reader to the facts we want to convey ... that it's easy to switch from Evernote.